docs/dev/table/streaming/joins.md

```diff
@@ -65,174 +65,147 @@ WHERE o.id = s.orderId AND
 
 Compared to a regular join operation, this kind of join only supports append-only tables with time attributes. Since time attributes are quasi-monotonic increasing, Flink can remove old values from its state without affecting the correctness of the result.
 
-Join with a Temporal Table Function
---------------------------
-
-A join with a temporal table function joins an append-only table (left input/probe side) with a temporal table (right input/build side),
-i.e., a table that changes over time and tracks its changes. Please check the corresponding page for more information about [temporal tables](temporal_tables.html).
+Temporal Joins
+--------------
+<span class="label label-danger">Attention</span> This feature is only supported in Blink planner.
+<span class="label label-danger">Attention</span> Attention it is only supported in SQL, and not supported in Table API yet.
```

Comment:
Aren't temporal joins still supported in the Table API using the old method (and listed [here](https://ci.apache.org/projects/flink/flink-docs-release-1.12/dev/table/tableApi.html#joins))? If so, this sentence is a bit misleading.

Reply:
I agree your point, @morsapaes . Old temporal joins(using temporal table function + `lateral`) is still supported in Table API, the newborn temporal join(using temporal table DDL + temporal join syntax) is supported in SQL.
How about :
```
Temporal table has ways to define in Flink, i.e. [temporal table function]({% link dev/table/streaming/temporal_tables.md %}#temporal-table-function) ) and [temporal table DDL](({% link dev/table/streaming/temporal_tables.md %}#temporal-table-ddl)).
Temporal join that using temporal table function is only supported in Table API,  temporal join that using temporal table DDL is only supported in SQL, please see the page [temporal table]({% link dev/table/streaming/temporal_tables.md %}) for more information about the differences between temporal table function and temporal table DDL.
```